src/components/molecules/StartTable/StartTable.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import classNames from "classnames";
+import { useAsyncFn } from "react-use";
+
+import { updateVenueTable } from "api/table";
+
+import { Table } from "types/Table";
+import { VenueTemplate } from "types/venues";
+
+import { currentVenueSelector } from "utils/selectors";
+
+import { useSelector } from "hooks/useSelector";
+
+import "./StartTable.scss";
+
+export interface StartTablePropsType {
+  tables: Table[];
+  newTable: Table;
+}
+
+export const StartTable: React.FC<StartTablePropsType> = ({
+  tables,
+  newTable,
+}) => {
+  const venue = useSelector(currentVenueSelector);
+
+  const [table] = tables;
+  const startTableWidth =
+    table.columns &&
+    venue?.template !== VenueTemplate.jazzbar &&
+    (table.columns + 1) * 55;
+  const startTableHeight =
+    table.rows &&
+    venue?.template !== VenueTemplate.jazzbar &&
+    table.rows * 50 + 65;
```

Comment:
export/import this hardcoded value from `settings.ts`

Reply:
fixed in 628728c978f8946518261612288a3324163ca176